### DISCUSSION TITLE
Satin stitch objects

### DISCUSSION THREAD

**Discussion by dhust** [inkstitch/inkstitch#1147] [Q&A]
Hi all. I'm trying to figure out how to print a logo using satin stitching. I tried using a fill but it doesn't look nearly as good. The letters in the logo were created using the pen tool. Our school sent out the logo to be turned into a embroidery fill and they were able to make it use satin stitch but there's a few issues. Instead of sending it out again and getting charged, I wanted to try to do it myself. Any advice would be greatly appreciated.

**Comment by tyrosinase** [comment]
There's some tips here on how to convert things to satin stitch: https://silverseams.com/2020/03/monogramming-without-a-font/

Note that since it's dealing with a small alphabet (probably *much* smaller than yours if you're dealing with  and mixing satin with running/bean, it doesn't address auto-routing satin. That's a tool that might or might not help depending on what the letters/logo look like.

You can also wander through the "freebies" tag and see how things like the knotwork shamrock look in SVG form, which may help: https://silverseams.com/tag/freebie (there are also conventional sewing and cutting-machine entries under that tag; pay them no mind).

**Comment by ghost** [comment]
Just a thought:
For lettering you can also use the lettering tool, unless you are looking for a specific font.  There will be more fonts available on the next release (I have been following).   I used TT masters.  This was on a clothing protector for a disabled High School student. My wife is a para at the high school. Excuse the wash away topping not yet washed away.

![IMG_20210329_125001710 (1)](https://user-images.githubusercontent.com/10312448/114411628-f93bfd00-9b71-11eb-8804-e43e267332c1.jpg)

**Reply by type8008** [reply]
how did you post a picture

**Comment by type8008** [comment]
I was Having the Same Problem, I don't Know if anyone knows about this workaround, but it works extremely well for me.  So, you prepare your whole project to the point of ready to save copy in, at that point what you can do is make sure that you put everything you want to be satin into 1 group. select the whole group, duplicate, don't click anywhere after you click duplicate.... go straight to extensions>ink stitch>edit>selection to pattern. And that is how you get a satin fill!

**Reply by type8008** [reply]
![roctron for github](https://user-images.githubusercontent.com/107273159/184719678-579bead8-f616-47d7-9d2e-1a22b20a58b5.jpg)

**Reply by type8008** [reply]
![satin for gityhub](https://user-images.githubusercontent.com/107273159/184719799-e94dfed5-a7f5-40a9-83c4-b243086a9746.jpg)
